Sources/Sentry/Public/SentryOptions.h

```diff
@@ -154,7 +154,8 @@ NS_SWIFT_NAME(Options)
 /**
  * Describes the Sentry SDK and its configuration used to capture and transmit an event.
  */
-@property (nonatomic, readonly, strong) SentrySdkInfo *sdkInfo;
+@property (nonatomic, readonly, strong) SentrySdkInfo *sdkInfo DEPRECATED_MSG_ATTRIBUTE(
```

Comment:
`m`: I think we should mention in the comment above that this is reserved for internal use, and developers shouldn't be using this.